ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/eventstream/services/EventStreamServiceImpl.java

```diff
@@ -92,11 +95,18 @@ public EventStreamResponse saveEventStream(EventStreamResponse eventStreamRespon
 		ShaclShape shaclShape = new ShaclShape(
 				eventStreamResponse.getCollection(),
 				eventStreamResponse.getShacl());
-		eventStreamCollection.saveEventStream(eventStream);
+		eventStreamRepository.saveEventStream(eventStream);
 		shaclShapeService.updateShaclShape(shaclShape);
 		if (eventStreamResponse.isDefaultViewEnabled()) {
 			viewService.addDefaultView(eventStream.getCollection());
 		}
 		return eventStreamResponse;
 	}
+
+	@EventListener(ApplicationStartedEvent.class)
+	public void initEventStream() {
```

Comment:
Shouldn't we publish a EventStreamChangedEvent in saveEventStream as well?

Reply:
Ha, correct!
I also noticed this. Picked-up in https://github.com/Informatievlaanderen/VSDS-LDESServer4J/pull/658